tests/attack/test_mod_wapp.py

```diff
@@ -0,0 +1,282 @@
+from unittest.mock import Mock
+
+import responses
+
+from wapitiCore.net.web import Request
+from wapitiCore.net.crawler import Crawler
+from wapitiCore.attack.mod_wapp import mod_wapp
+
+class FakePersister:
+    def __init__(self):
+        self.requests = []
+        self.additionals = []
+        self.anomalies = set()
+        self.vulnerabilities = set()
+
+    def get_links(self, _path=None, _attack_module: str = ""):
+        return self.requests
+
+    def add_additional(self, request_id: int = -1, category=None, level=0, request=None, parameter="", info=""):
+        self.additionals.append(info)
+
+    def add_anomaly(self, _request_id: int = -1, _category=None, _level=0, _request=None, parameter="", _info=""):
+        self.anomalies.add(parameter)
+
+    def add_vulnerability(self, _request_id: int = -1, _category=None, _level=0, _request=None, parameter="", _info=""):
```

Comment:
If you dont use a variable, it doesnt need a default value `_request_id: int = -1` -> `_request_id`